src/main/java/xbot/common/injection/electrical_contract/XDeadwheelElectricalContract.java

```diff
@@ -0,0 +1,11 @@
+package xbot.common.injection.electrical_contract;
+
+import edu.wpi.first.units.measure.Distance;
+
+/**
+ * This interface defines the base electrical contract
+ * for robots implementing a differential drive based system.
+ */
+public interface XDeadwheelElectricalContract {
+    public abstract Distance getDistanceFromCenterToOuterBumperX();
```

Comment:
❓ is this really about the deadwheels or something that should just live on every robot?

Reply:
The base electrical contract already has `getDistanceFromCenterToOuterBumperX`, why is it needed here as well?